team-builder/src/Component/Form.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+
+const Create = props => {
+  const reset = { name: "", email: "", role: "" };
+  const [newCard, setNewCard] = useState(reset);
+
+  const handleChange = event => {
+    setNewCard({
+      ...newCard,
+      [event.target.name]: event.target.value
```

Comment:
Can you explain to me what the brackets around the `event.target.name` does?  What would happen if we wrapped `event.target.value` in brackets?  What would happen?